app/models/host_aggregate.rb

```diff
@@ -0,0 +1,43 @@
+class HostAggregate < ApplicationRecord
+  include NewWithTypeStiMixin
+  include Metric::CiMixin
+  include EventMixin
+  include ProviderObjectMixin
+  include VirtualTotalMixin
+
+  acts_as_miq_taggable
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+
+  # FIXME: I don't think this is needed here
```

Comment:
I haven't done anything with rollups yet since I'm unclear exactly what the requirements are either. 
@tzumainn who would know what the rollups requirements are for host aggregates?